src/adapters/github.less

```diff
@@ -179,6 +179,8 @@
         outline: none;
         padding-right: 0;
         text-decoration: none;
+        width: calc(100% - 20px);
```

Comment:
I suppose file name will be cut off. Should we show ...?

Reply:
@buunguyen  Done a. It's good to go.